### PR TITLE
Handle imports in <body>

### PR DESCRIPTION
**Pull request by azakus**
- Make sure import head contents are in head and import body contents are in
  body
- Make sure import body contents are at the top of body
- Clean up a bit more leftover whitespace when removing imports in head and body

### CODE REVIEW

File: lib/vulcan.js [hunk 1]
```diff
@@ -42,10 +42,26 @@ function isExcludedImport(importMeta) {
   });
 }
 
-function replaceWith(parent, node, replacements) {
-  var idx = parent.childNodes.indexOf(node);
-  parent.childNodes = parent.childNodes.slice(0, idx).
-    concat(replacements, parent.childNodes.slice(idx + 1));
+function isBlankTextNode(node) {
+  return node && dom5.isTextNode(node) && !/\S/.test(dom5.getTextContent(node));
```
Comment (nevir) [inline lib/vulcan.js]: double negation, you heathen!

`/\s/.test(dom5.getTextContent(node))`

Comment (azakus) [inline lib/vulcan.js]: No, that would be if any character was a whitespace.
`!/\S/` is if any character *isn't* whitespace.

Comment (nevir) [inline lib/vulcan.js]: I rescind this!

**Comment by nevir** [comment]
LGTM

**Pull request merged**